pennylane/qchem/basis_data.py

```diff
@@ -20,18 +20,28 @@
 
 import itertools
 
+# fmt: off
```

Comment:
Please remove this.

Reply:
These markers prevent black from turning this into a 108 lines dictionary.

Reply:
Please leave a quick comment saying something like "added to prevent reformatting by black" to be safe.